src/main/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteDeleteOrderRequestHandler.java

```diff
@@ -23,6 +23,8 @@ public RemoteDeleteOrderRequestHandler() {
 
     @Override
     public IQ handle(IQ iq) {
+        LOGGER.info("Received request for order: " + iq.getID());
```

Comment:
I am not sure but it seems line 42 should be a catch for throwable. Please, talk to @wassoares @arn3tt or @GustavoYamaguchi